### PR TITLE
RD-2078 Deploy On test fix

### PR DESCRIPTION
**Pull request by kubama**
The reason of the failures was a timing issue when selecting values from dynamically loaded dropdown (Blueprint field in this case).
As a fix and also to prevent such issues in the future I created ` setDropdownValue` command that does the thing in a safe way.

It is possible to reproduce the issue manually if you type fast. However, this is such an edge case that I don't consider it worth addressing as a DynamicDropdown component fix.

System tests all green on the branch: https://jenkins.cloudify.co/view/UI/job/Stage-UI-System-Test/627/

### CODE REVIEW

File: test/cypress/integration/widgets/deployments_view_spec.ts [hunk 1]
```diff
@@ -865,7 +865,7 @@ describe('Deployments View widget', () => {
             const labelKey = 'label_key';
             const labelValue = 'label_value';
             cy.get('.modal').within(() => {
-                cy.contains('.field', 'Blueprint').find('input').type(`${blueprintName}{enter}`);
+                cy.setDropdownValue('Blueprint', blueprintName);
```
Comment (qooban) [inline test/cypress/integration/widgets/deployments_view_spec.ts:868]: Could you also apply the same solution for workflow selection in "should allow to run workflow on filtered deployments" test in this spec?

**Review by qooban — COMMENTED** [review]
Added one comment.

File: test/cypress/integration/widgets/deployments_view_spec.ts (hunk 1 above)
Comment (kubama) [reply test/cypress/integration/widgets/deployments_view_spec.ts:868]: Good point, applied

**Review by qooban — APPROVED** [review]

**Comment by kubama** [comment]
Since this is a simple change I'm merging it with just one approval

**Review by Gelio — COMMENTED** [review]
Nice :ok_hand:
